methods_and_flow_control/fizz_buzz.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
```

Comment:
I see all four learning_by_doing subdirectories in this branch, why is the one .rb showing up in this commit?